modules/shortcodes/vimeo.php

```diff
@@ -172,6 +172,77 @@ function vimeo_shortcode( $atts ) {
 
 add_shortcode( 'vimeo', 'vimeo_shortcode' );
 
+/**
+ * Filters the Vimeo shortcode to be AMP-compatible.
+ *
+ * @param string $html The shortcode HTML.
+ * @param string $shortcode_tag The shortcode's tag (name).
+ * @param array  $attr The attributes of the shortcode.
+ * @return string The filtered HTML.
+ */
+function amp_vimeo_shortcode( $html, $shortcode_tag, $attr ) {
+	if (
+		( class_exists( 'Jetpack_AMP_Support' ) && ! Jetpack_AMP_Support::is_amp_request() )
+		||
+		'vimeo' !== $shortcode_tag
+	) {
```

Comment:
If `Jetpack_AMP_Support` does not exist at all, we'll end up continuing with the function. Ideally we want to cover 3 use-cases:

- A different shortcode tag.
- No `Jetpack_AMP_Support` class.
- Not an AMP request.

If any of those is true, we bail.

Reply:
Sounds good, https://github.com/Automattic/jetpack/pull/13921/commits/4e3a04708514fa918bc75de6ab67b0882e19754c changes this so that it returns if any of those 3 cases is true.